pkg/wasm/cache.go

```diff
@@ -90,7 +90,7 @@ type cacheEntry struct {
 // NewLocalFileCache create a new Wasm module cache which downloads and stores Wasm module files locally.
 func NewLocalFileCache(dir string, purgeInterval, moduleExpiry time.Duration, insecureRegistries []string) *LocalFileCache {
 	cache := &LocalFileCache{
-		httpFetcher:        NewHTTPFetcher(),
+		httpFetcher:        NewHTTPFetcher(5 * time.Second),
```

Comment:
Should this be user configurable via an environment variable? 5s seems reasonable for most cases, but may not be suitable for some users.

Reply:
I totally agree with the comment.

Currently, many variables are hard-coded regarding `cache` and `fetcher`. 
For example, at the top of pkg/wasm/cache.go, we can see `DefaultWasm.*` const and some of them seems to be configurable. 

One of the problem here is, if we want to make them configurable, then one of the functions of cache should take a long list of parameters, and it may causes poor readability. So, I am thinking about refactoring PR which is including `cache.go` as well as `imagefetcher.go`/`httpfetcher.go`.

Therefore, I would like to suggest to push making them configurable back, and at now, just keep the timeout per request and the number of retries as the constants.
(Please note that I pushed a commit to use the `DefaultWasmHTTPRequestTimeout` constant instead of directly using `5 * time.Second`.)

WDYT?